script/tool/lib/src/common/package_looping_command.dart

```diff
@@ -167,4 +195,80 @@ abstract class PackageLoopingCommand extends PluginCommand {
     }
     print(Colorize(heading)..cyan());
   }
+
+  /// Prints a summary of packges run, packages skipped, and warnings.
+  void _printRunSummary(List<Directory> packages) {
+    final int skipCount = _skippedPackages.length;
+    // Split the warnings into those from packages that ran, and those that
+    // were skipped.
+    final Set<Directory> _skippedPackagesWithWarnings =
+        _packagesWithWarnings.intersection(_skippedPackages);
+    final int skippedWarningCount = _skippedPackagesWithWarnings.length;
+    final int runWarningCount =
+        _packagesWithWarnings.length - skippedWarningCount;
+
+    final String runWarningSummary =
+        runWarningCount > 0 ? ' ($runWarningCount with warnings)' : '';
+    final String skippedWarningSummary =
+        runWarningCount > 0 ? ' ($skippedWarningCount with warnings)' : '';
+    print('------------------------------------------------------------');
+    if (hasLongOutput) {
```

Comment:
Is `hasLongOutput` suppose to mean "verbose" in this case?  We had another place in the code where the same verbiage meant "compress the output because it's long" I thought.

Reply:
It means that in practice, the output of a given `runForPackage` call is likely to be long. The existing use is that it changes the header format from a single line to the ====-wrapped banner style, to make it easier to skim for a run break.

Reply:
(I deliberately called it "long" rather than "verbose" because I don't want to imply that underlying commands are being called with a `-v` or similar flag; some commands print a lot even when you don't make them explicitly verbose.)